### ISSUE TITLE
The initial value cannot be displayed when routing is switched with vue-route.

### ISSUE THREAD

**Issue by s952787097** [tinymce/tinymce-vue#27]
> vue            V2.5.16
> chrome      V68

Use as Example
```
<edit :init="init" v-model="content"></edit>

data(){
       return {
             content: '123';
       }
}
```

Then switch vue-route.
```
import Wikis from './views/Wikis.vue';
import AddWiki from './views/AddWiki.vue';

export default {
    routes: [      
        {
            name: 'wikis',
            path: '/Wikis/',
            component: Wikis
        },
        {
            name: 'addwiki',
            path: '/AddWiki/',
            component: AddWiki
        }
    ]
};
```
Content can be displayed at AddWiki initialization, after switching to Wikis and then to AddWiki, Content is not displayed

**Comment by fyrkant** [comment]
I'm not sure I understand the problem or how that can fix it, can you create a demo app showing the error and maybe I can find a better way to fix it? I don't think you should have to set content with format raw.

**Comment by chitthaenaing** [comment]
@s952787097  I think this issue has already been solved on version 1.0.8. 
You can check it out at 804d4993d2ea386407225f791321b694b62e6dec

**Comment by fyrkant** [comment]
Closing this because of inactivity.
